spec.html

```diff
@@ -23761,6 +24667,13 @@ <h1>EvalDeclarationInstantiation ( _body_, _varEnv_, _lexEnv_, _strict_ )</h1>
                     1. NOTE: Annex <emu-xref href="#sec-variablestatements-in-catch-blocks"></emu-xref> defines alternate semantics for the above step.
                   1. NOTE: A direct eval will not hoist var declaration over a like-named lexical declaration.
               1. Set _thisEnv_ to _thisEnv_.[[OuterEnv]].
+          1. Let _privateIdentifiers_ be a new empty List.
+          1. Let _privEnv_ be _privateEnv_.
```

Comment:
I am not a fan of having both `_privEnv_` and `_privateEnv_`.  Can we call this `_pointer_` or something?